src/data/locales/en/translation.json

```diff
@@ -399,7 +399,7 @@
   "generic-validation-error-message": "There are one or more errors on this page. Review and correct them to continue. Complete all fields and try again.",
   "timeout-modal": {
     "header": "Your Session Will End Soon",
-    "warning": "To protect your information, you will be logged out in 5 minutes if you do not continue working. Any unsaved changes will be lost.",
+    "warning": "To protect your information, you will be logged out in {{numberOfMinutes}} minutes if you do not continue working. Any unsaved changes will be lost.",
```

Comment:
Do we care that it will say 1 minute**s**? I imagine i18n-next has some handling of plurals, but maybe it's not worth the effort.

Reply:
Yeah, I'll update it if EDD requests a change. There is support for plurals in the library, but we'd need an updated Spanish translation.